### PR TITLE
add includeKeys convenience to PFQuery

### PR DESCRIPTION
**Pull request by AndrewSB**
Thought this was a useful method

### CODE REVIEW

**Comment by nlutsenko** [comment]
I think it is, indeed.
Let's do a little bit of work to improve it then...
Pieces that would be required to get this merged in:
- Adding the method to public header file
- Adding documentation on the method in the public header
- Specifying the generic type of the `NSArray *` to be `NSArray<NSString *> *`, since it adds more type safety.
- Adding `includeKeys:` method to `PFMutableQueryState` and using it instead of calling `includeKey:` in a loop, to slightly improve performance (this is nitpicking)

Let me know if you need help with any of these.

**Comment by AndrewSB** [comment]
Sorry about not thinking to do all of that myself 😬.

Haven't done any real Objective-C work in about 18 months. I'll make those changes in the next couple days and ping you when I think it's ready to go 👌

**Comment by AndrewSB** [comment]
@nlutsenko would you like me to also squash all of my commits?

**Comment by nlutsenko** [comment]
Yes, please.

File: Parse/PFQuery.h [hunk 1]
```diff
@@ -95,6 +95,15 @@ typedef void (^PFQueryArrayResultBlock)(NSArray<PFGenericObject> *_Nullable obje
 - (instancetype)includeKey:(NSString *)key;
 
 /**
+ Make the query include PFObjects that have a reference stored at the provided keys.
```
Comment (nlutsenko) [inline Parse/PFQuery.h]: Nit: `PFObject`s

**Comment by nlutsenko** [comment]
A single nit, otherwise looks great!
Squash - yes, please, if it's not a problem.

File: Parse/PFQuery.h [hunk 2]
```diff
@@ -83,7 +83,7 @@ typedef void (^PFQueryArrayResultBlock)(NSArray<PFGenericObject> *_Nullable obje
 ///--------------------------------------
 
 /**
- Make the query include PFObjects that have a reference stored at the provided key.
+ Make the query include `PFObject`s that have a reference stored at the provided key.
```
Comment (AndrewSB) [inline Parse/PFQuery.h]: added a "`" to the docs for includeKey as well

Comment (nlutsenko) [inline Parse/PFQuery.h]: Nice catch, thank you!

**Pull request merged**